docs/docs/hpc-on-k8s/minio.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Minio in Kubernetes
+
+To deploy Minio in our Kubernetes cluster, we utilize Helm charts. Prior to installation, it's crucial to ensure that the Helm CLI is properly set up and the kubeconfig file is located in the designated directory.
+
+For this installation, we've opted for the default namespace, employing the following configurations:
+
+- storageClass:  
+- Minio rootUser
+- Minio rootPassword
+- Minio ServiceType
+- Minio API port
+- Minio console port
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+helm install -n default  --set global.storageClass=longhorn --set auth.rootUser=admin --set auth.rootPassword=test123456 --set service.type=NodePort --set service.nodePorts.api=31100 --set service.nodePorts.console=31101 --set persistence.size=2Gi my-minio bitnami/minio --version 12.9.4
```

Comment:
I know that there were some issues in testing where the previous 8Gi persistent size was an issue but just changing it to 2 could also bring up issues with file storage. 

Perhaps there should be a note or some information about considerations and constraints for the storage size.

Reply:
updated thanks